client/src/components/ProductInformation.jsx

```diff
@@ -1,18 +1,33 @@
 import React from 'react';
 import StarReview from './StarReview.jsx';
+import PriceLine from './PriceLine.jsx';
+
+const calculateRating = function(ratings) {
```

Comment:
This is a useful function that several components need. We should modularize it if/when we have refactor time.